content/browser/android/content_startup_flags.cc

```diff
@@ -87,6 +87,12 @@ void SetContentCommandLineFlags(bool single_process) {
   // Prevents the renderer process from being killed for WebRTCGameStreaming.
   parsed_command_line->AppendSwitch(
       service_manager::switches::kNoSandbox);
+
+  // For using Knox in the WebRTC Service, permission must be passed to the
+  // renderer process. Currently, the Knox permission only applies to the
+  // browser process and it cannot be propagated to the renderer process.
+  // Therefore we apply single process mode for Knox system input.
+  parsed_command_line->AppendSwitch(switches::kSingleProcess);
```

Comment:
The single process mode should be enabled if |enable_service_offloading_knox| is true.

Reply:
Thank you. Done